File: app/src/main/java/ru/roscha_akademii/medialib/viewvideo/presenter/ShowVideoPresenter.java
package ru.roscha_akademii.medialib.viewvideo.presenter;

import com.hannesdorfmann.mosby.mvp.MvpPresenter;

import ru.roscha_akademii.medialib.viewvideo.view.ShowVideoView;

public interface ShowVideoPresenter extends MvpPresenter<ShowVideoView> {
    void start(long videoId);
}

File: app/src/main/java/ru/roscha_akademii/medialib/viewvideo/presenter/ShowVideoPresenter.kt
package ru.roscha_akademii.medialib.viewvideo.presenter

import com.hannesdorfmann.mosby.mvp.MvpPresenter

import ru.roscha_akademii.medialib.viewvideo.view.ShowVideoView

interface ShowVideoPresenter : MvpPresenter<ShowVideoView> {
    fun start(videoId: Long)
}
